echopype/convert/set_groups_ek80.py

```diff
@@ -906,7 +924,7 @@ def set_vendor(self) -> xr.Dataset:
         #  Save decimation factors and filter coefficients
         coeffs = dict()
         decimation_factors = dict()
-        for ch in self.parser_obj.ch_ids["power"] + self.parser_obj.ch_ids["complex"]:
+        for ch in channels:
```

Comment:
(this is a note to myself to double check, but I think the implementation is correct)
It is not clear to me why we had the `ch_ids["power"]` + `ch_ids["complex"]` before, but the union of these two should just be `channels` as implemented in this PR.